src/Microsoft.CodeAnalysis.Analyzers/Core/xlf/CodeAnalysisDiagnosticsResources.cs.xlf

```diff
@@ -4,427 +4,427 @@
     <body>
       <trans-unit id="AddEntryForDiagnosticIdInAnalyzerReleaseCodeFixTitle">
         <source>Add rule entry to unshipped release file</source>
-        <target state="translated">Přidat položku pravidla k nevydanému souboru verze</target>
+        <target state="new">Add rule entry to unshipped release file</target>
```

Comment:
⚠️ ⚠️ This removes all the translations.

Reply:
Please close these PRs. I will report this issue to the loc team and it will be fixed shortly.